File: LC2703-ReturnLenOfArg/solution.go
package main

import "fmt"

func argumentsLength(args ...int) int {
	return len(args)
}

func main() {
	// Call the argumentsLength function with arguments
	length := argumentsLength(112, 212, 100, 45)
	fmt.Println(length) // Output: 3
}

File: LC2703-ReturnLenOfArg/solution.js

/** 
* @return {number}
*/

//declare a function named argumentsLength. The ...args is called a "rest parameter", which collects all the arguments passed to the function into an array named args.
var argumentsLength = function(...args) {
    //returns the length of the args array, which represents the number of arguments passed to the function.
    //used as the member access operator. It allows you to access properties and methods of objects.
    //to access methods and properties of other objects. ( . )
    return args.length;
};

// /**
// * arugmentsLength(1, 2, 3);
//  */